<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USBSettingsWidget</class>
 <widget class="QWidget" name="USBSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="selectBox">
       <property name="title">
        <string>Select Serial Port</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="manufacturerLabel">
          <property name="text">
           <string>Manufacturer:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="pidLabel">
          <property name="text">
           <string>Product ID:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="serialNumberLabel">
          <property name="text">
           <string>Serial number:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="serialPortInfoListBox"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="vidLabel">
          <property name="text">
           <string>Vendor ID:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="locationLabel">
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
